dts/arm/st/f0/stm32f0.dtsi

```diff
@@ -165,6 +166,15 @@
 			label = "IWDG";
 		};
 
+		wwdg: watchdog@40002c00 {
+			compatible = "st,stm32-watchdog";
```

Comment:
I don't think we should reuse same compatible as IWDG.
These are not similar IPs and hence can't share same 'compatible'.
Also, if used together (which is a valid use case), this will raise issue in fixup's handling.

Reply:
Done